logviewer/src/main/java/com/nodesagency/logviewer/screens/logs/LogsFragment.kt

```diff
@@ -52,6 +51,29 @@ internal class LogsFragment : Fragment() {
             .get(LogEntriesViewModel::class.java)
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?, inflater: MenuInflater?) {
+        super.onCreateOptionsMenu(menu, inflater)
+        inflater?.inflate(R.menu.menu_logs, menu)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem?): Boolean {
+       return when (item?.itemId) {
+            R.id.logsCategoryShare -> {
+                val shareMessage = "Logs from ${category.name} category\n" +
+                        logEntriesViewModel.logEntryList.value?.joinToString("\n") {it.toShareMessage()}
+                val sendIntent: Intent = Intent().apply {
+                    action = Intent.ACTION_SEND
+                    putExtra(Intent.EXTRA_TEXT, shareMessage)
+                    type = "text/plain"
+                }
+                startActivity(Intent.createChooser(sendIntent, "Share logs category"))
```

Comment:
Could you move this string to `strings.xml`, please? It would be good to keep this consistent for potential localizations later.